src/Symfony/Bridge/Twig/Command/DebugCommand.php

```diff
@@ -242,4 +247,63 @@ private function getPrettyMetadata($type, $entity)
             return $meta ? '('.implode(', ', $meta).')' : '';
         }
     }
+
+    private function findWrongBundleOverrides($loaderPaths)
+    {
+        $alternatives = array();
+        $paths = array_unique($loaderPaths['(None)']);
+        $bundleNames = array();
+        foreach ($paths as $path) {
+            $relativePath = $path.'/bundles/';
```

Comment:
You don't need to check all paths because only `<twig.default_path>/bundles/...` is taking into account:
https://github.com/symfony/symfony/blob/6bbb5bcc5294fdd540bec35541f77b87e212fc5f/src/Symfony/Bundle/TwigBundle/DependencyInjection/TwigExtension.php#L170-L172

Use the `%twig.default_path%` parameter instead.

Reply:
and probably [`<kernel.root_dir>/<BundleName>/views`](https://github.com/symfony/symfony/blob/6bbb5bcc5294fdd540bec35541f77b87e212fc5f/src/Symfony/Bundle/TwigBundle/DependencyInjection/TwigExtension.php#L165-L167) should be hardcoded.

Reply:
For your first comment, I had the same idea but I did this choice for two reasons :
- I don't check all paths, just generic ones
- I prefered to use the result of other methods in the command that to duplicate code from another bundle.

Could you please tell me why is better to use the default path parameter?
Can someone give me another point of vue on it?

Reply:
Done !